### DISCUSSION TITLE
Compiling binaries

### DISCUSSION THREAD

**Discussion by Axilotl17** [ppy/osu#21193] [Q&A] [answered]
is there a way i can compile the code to a binary from the code without waiting for a release; like, creating `
osu.AppImage` with just the code

**Comment by smoogipoo** [comment, marked as answer]
Read the README.

**Reply by Axilotl17** [reply]
I have alot of issues when i do that; most maps are broke, with incorrect files. i copied over much of the files from my original installation, but everything seems to be broken.

**Reply by smoogipoo** [reply]
Dunno, sorry. I don't know what you've done/how you've tried to set it up, and you've provided no logs, so the rest is in your hands.

**Reply by bdach** [reply]
Also if you're not sure what you're doing you shouldn't probably be copying over internal game files. The dev builds have separate storage for good reasons.
